charts/tidb-cluster/templates/config/_tidb-config.tpl

```diff
@@ -59,7 +59,11 @@ format = "text"
 disable-timestamp = false
 
 # Stores slow query log into separated files.
+{{- if .Values.tidb.separateSlowLog }}
+slow-query-file = "/tmp/log/tidb/slowlog"
```

Comment:
This also needs to be changed to `/var/log/tidb/slowlog`.

Reply:
Good catch.
BTW, this is kind of a code smell but I can't figure out a better solution. I'll document this caveat or any better ideas?

Reply:
I mean user may override the configuration by setting `tidb.config`

Reply:
I think we can specify the command line argument `-log-slow-query` of tidb-server, Its priority is higher than the configuration file.